gdk/src/wayland/wayland_device.rs

```diff
@@ -0,0 +1,31 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(any(feature = "v3_22", feature = "dox"))]
+use crate::DevicePadFeature;
+#[cfg(any(feature = "v3_22", feature = "dox"))]
+use glib::translate::*;
+
+glib::wrapper! {
+    #[doc(alias = "GdkWaylandDevice")]
+    pub struct WaylandDevice(Object<ffi::GdkWaylandDevice>) @extends crate::Device;
+
+    match fn {
+        type_ => || ffi::gdk_wayland_device_get_type(),
+    }
+}
+
+impl WaylandDevice {
```

Comment:
`gdk_wayland_device_get_node_path` is missing. There are also a bunch of other wayland specific functions which could be manually binded as well; see gdk4-wayland

Reply:
I don't think so? I looked at the gtk repository directly for all wayland functions and I didn't see any others...

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gdk/wayland/gdkwaylanddevice.h#L58